src/NexusMods.App.UI/LeftMenu/Items/IApplyControlViewModel.cs

```diff
@@ -5,10 +5,16 @@ namespace NexusMods.App.UI.LeftMenu.Items;
 public interface IApplyControlViewModel : IViewModelInterface
 {
     ICommand ApplyCommand { get; }
+    
+    ICommand IngestCommand { get; }
 
     bool CanApply { get; }
+    
+    bool CanIngest { get; }
 
     bool IsApplying { get; }
+    
+    bool IsIngesting { get; }
```

Comment:
You don't really need `CanApply`, `CanIngest`, `IsApplying` and `IsIngesting`. All of those properties are included in `ReactiveCommand`. Changing `ICommand` to `ReactiveCommand<Unit, Unit>` should allow you to remove the other properties.

Reply:
Can you hide the button with reactive command? Because I need to hide the apply button to show the Applying spinner

Reply:
`ReactiveCommand` has `CanExecute` and `IsExecuting` properties that you can subscribe to.

Reply:
I'll remove IsExecuting, which is a duplicate. I guess you want to use CanExecute and bind it to IsVisible?

Reply:
Yeah, you essentially duplicated the behavior with those properties.

Reply:
I'll try to clean that up

Reply:
Had to fight with it for a while, as it wasn't behaving like expected. Should be working right now